npm/private/npm_import.bzl

```diff
@@ -415,6 +415,8 @@ def _impl(rctx):
         version = rctx.attr.version,
     ))
 
+    rctx_files["BUILD.bazel"].append(rctx.attr.extra_build_content)
```

Comment:
perhaps
```
if rctx.attr.extra_build_content:
```
then append a new line first

Reply:
The _JS_PACKAGE_TMPL happens to end with newline, so we don't need one, but you're right we may as well be defensive here